src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -932,6 +932,173 @@ public static void GetFullPath_Windows_Wildcards(char wildcard)
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareG", @"\\.\UNC\LOCALHOST\shareG\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareH\dir", @"\\.\UNC\LOCALHOST\shareH\dir\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\shareK\", @"\\.\UNC\LOCALHOST\shareK\foo" };
+            yield return new object[] { "foo", @"\\.\UNC\LOCALHOST\  shareL\", @"\\.\UNC\LOCALHOST\  shareL\foo" };
+
+            // Current drive rooted
+            yield return new object[] { @"\tmp\bar", curDir, Path.GetPathRoot(curDir)+ @"tmp\bar" };
+            yield return new object[] { @"\.\bar", curDir, Path.GetPathRoot(curDir) + @"bar" };
+            yield return new object[] { @"\tmp\..", curDir, Path.GetPathRoot(curDir).Substring(0,2)};
+            yield return new object[] { @"\tmp\bar\..", curDir, Path.GetPathRoot(curDir) + @"tmp" };
+            yield return new object[] { @"\tmp\bar\..", curDir, Path.GetPathRoot(curDir) + @"tmp" };
+            yield return new object[] { @"\", curDir, Path.GetPathRoot(curDir) };
+
+            // Specific drive rooted
+            yield return new object[] { Path.GetPathRoot(curDir)[0] + @":tmp\foo\..", curDir, Path.GetPathRoot(curDir) + @"tmp" };
+            yield return new object[] { Path.GetPathRoot(curDir)[0] + @":tmp\foo\.", curDir, Path.GetPathRoot(curDir) + @"tmp\foo" };
+            yield return new object[] { Path.GetPathRoot(curDir)[0] + @":tmp\foo\..", curDir, Path.GetPathRoot(curDir) +@"tmp"  };
+            yield return new object[] { Path.GetPathRoot(curDir)[0] + @":tmp", curDir, Path.GetPathRoot(curDir) + @"tmp" };
+            yield return new object[] { Path.GetPathRoot(curDir)[0] + @":", curDir, Path.GetPathRoot(curDir) };
+            yield return new object[] { Path.GetPathRoot(curDir)[0], curDir, curDir + Path.DirectorySeparatorChar + Path.GetPathRoot(curDir)[0] };
+
+            yield return new object[] { @"Z:tmp\foo\..", curDir, @"Z:\tmp" };
+            yield return new object[] { @"Z:tmp\foo\.", curDir, @"Z:\tmp\foo" };
+            yield return new object[] { @"Z:tmp\foo\..", curDir, @"Z:\tmp" };
+            yield return new object[] { @"Z:tmp", curDir, @"Z:\tmp" };
+            yield return new object[] { @"Z:", curDir, @"Z:\" };
+            yield return new object[] { @"Z", curDir, curDir + Path.DirectorySeparatorChar + @"Z" };
+        }
+
+        [Theory]
+        [MemberData(nameof(GetFullPath_BasePath_BasicExpansions_TestData))]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public static void GetFullPath_BasicExpansions(string path, string basePath, string expected)
+        {
+            Assert.Equal(expected, Path.GetFullPath(path, basePath));
+        }
+
+
+        [Fact]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        public static void GetFullPath_BasePath_InvalidInput_Unix()
```

Comment:
This should be moved up next to the corresponding Windows one.  The Unix one also has "Unix" in the name but the Windows one lacks "Windows"; they should be consistent.